recipe/recipe.yaml

```diff
@@ -0,0 +1,63 @@
+context:
+  name: "awkward-cpp"
+  version: "46"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/${{ name|replace('-', '_') }}-${{ version }}.tar.gz
+  sha256: b82becb65a33280c6c27336ee5c037c5b7e394be52565aa31e087ca54adce59e
+
+build:
+  number: 1
+  variant:
+    use_keys:
+      - x86_64_microarch_level
+    down_prioritize_variant: ${{ 0 if x86_64_microarch_level == 1 else 1 }}
+  script: ${{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
+    - if: unix and x86_64
+      then: x86_64-microarch-level ==${{ x86_64_microarch_level }}
```

Comment:
TIL. My understanding had been that while `<package name> ==<package version>` is valid, the preferred syntax is `<package name> <package version>`, but then I went and checked the `rattler-build` docs and you see this pattern instead. https://rattler.build/latest/reference/recipe_file/#run-constraints

:+1: Good to know.

Reply:
Yeah it was giving me build errors when I tried the way you mentioned. The documentation skew is a bit rough.